Lib/test/test_aifc.py

```diff
@@ -145,6 +146,20 @@ class AifcALAWTest(AifcTest, unittest.TestCase):
 
 
 class AifcMiscTest(audiotests.AudioTests, unittest.TestCase):
+
+    @mock.patch("aifc.open")
+    def test_openfp_deprecated(self, mock_open):
+        arg = "arg"
+        mode = "mode"
+        with warnings.catch_warnings(record=True) as w:
+            warnings.simplefilter("always")
+            aifc.openfp(arg, mode=mode)
+
+            self.assertTrue(len(w) == 1)
+            self.assertTrue(issubclass(w[0].category, DeprecationWarning))
```

Comment:
Style nit: `assertInstance` might be better option here.

Reply:
I initially thought that as well, but `category` isn't an instance there.

The other comments look good and I'll push a fix for them.

Reply:
By using `assertWarns` we get the behavior we needed from those two `assertTrue` lines, so I just removed them in ebe685d.